protopipe/scripts/data_training.py

```diff
@@ -222,13 +223,27 @@ def main():
     outfile = tb.open_file(args.outfile, mode="w")
     outTable = {}
     outData = {}
+    
+    # Configuration options for SimTelEventSource
+    # Readout window integration correction
+    try:
+        calib_scale = cfg["Calibration"]["calib_scale"]
+    except KeyError:
+        # defaults for no calibscale applied
+        calib_scale = 1.0
+
+    cfg_SimTelEventSource = Config()
+    cfg_SimTelEventSource.MySimTelEventSource.calib_scale = calib_scale
```

Comment:
you can also just use this as an argument when constructing it below (making a Config is only really needed for reading external config files)

Reply:
done